packages/calling-stateful-client/src/CallSubscriber.ts

```diff
@@ -248,14 +256,26 @@ export class CallSubscriber {
   /* @conditional-compile-remove(close-captions) */
   private initCaptionSubscriber = (): void => {
     // subscribe to captions here so that we don't call captions when call is not initialized
-    if (this._call.state === 'Connected' && !this._captionsSubscriber) {
+    if (
+      this._call.state === 'Connected' &&
+      !this._TeamsCaptionsSubscriber &&
+      /* @conditional-compile-remove(acs-close-captions) */ !this._CaptionsSubscriber
```

Comment:
asking in chat but commenting here so I don't forget, can it switch between _CaptionsSubscriber and _TeamsCaptionsSubscriber if a teams caption bot is pulled into the call?